app/assets/images/payment_icons/swish.svg

```diff
@@ -1 +1 @@
lor="#f17345"/><stop offset=".663" stop-color="#f00912"/><stop offset=".878" stop-color="#ee0920"/><stop offset="1" stop-color="#e30767"/></radialGradient><path fill-rule="evenodd" clip-rule="evenodd" fill="url(#SVGID_1_)" d="M25.239 15.231c.025-.241.037-.487.037-.737 0-.168-.009-.334-.024-.495a5.666 5.666 0 00-.049-.544c-.209-1.489-.868-2.79-1.979-3.901a7.385 7.385 0 00-1.389-1.1l1.545-1.92c.193.153.382.322.567.507a6.864 6.864 0 011.291 1.727c.509.975.762 2.05.762 3.226 0 .354-.02.701-.061 1.039a6.87 6.87 0 01-.7 2.198z"/><radialGradient id="SVGID_2_" cx="22.283" cy="-1.836" r="10.993" gradientTransform="matrix(1.0345 0 0 1.0351 -.648 10.826)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#fdd626"/><stop offset=".314" stop-color="#eb5253"/><stop offset=".506" stop-color="#eb4f65"/><stop offset=".639" stop-color="#e739cb"/><stop offset=".8" stop-color="#9e16ec"/><stop offset="1" stop-color="#5702f0"/></radialGradient><path fill-rule="evenodd" clip-rule="evenodd" fill="url(#SVGID_2_)" d="M21.836 8.455c.49.298.954.665 1.389 1.1 1.11 1.111 1.77 2.412 1.979 3.901.024.178.041.359.049.544.016.161.024.327.024.495 0 .25-.012.496-.037.737a7.633 7.633 0 01-1.291 1.728C22.58 18.319 20.932 19 19 19a6.93 6.93 0 01-2.728-.531 5.773 5.773 0 001.882.302c1.601 0 2.969-.567 4.103-1.703.379-.378.697-.785.954-1.22.032-.064.064-.125.097-.182.435-.813.651-1.715.651-2.705 0-1.603-.567-2.972-1.702-4.107a9.35 9.35 0 00-.421-.399z"/><radialGradient id="SVGID_3_" cx="15.33" cy="3.624" r="10.84" gradientTransform="matrix(1.0345 0 0 1.0354 -.648 10.826)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#424ee9"/><stop offset=".392" stop-color="#58c9be"/><stop offset=".765" stop-color="#6cd82e"/><stop offset="1" stop-color="#e5db2a"/></radialGradient><path fill-rule="evenodd" clip-rule="evenodd" fill="url(#SVGID_3_)" d="M16.164 15.533a7.244 7.244 0 01-1.255-1.015c-1.159-1.159-1.827-2.521-2.004-4.083a2.762 2.762 0 01-.024-.362 5.024 5.024 0 01-.024-.496c0-.386.027-.757.084-1.111a7.08 7.08 0 011.11-1.426C15.42 5.681 17.069 5 19 5c1.07 0 2.052.206 2.944.616a5.926 5.926 0 00-2.063-.351c-1.601 0-2.969.564-4.103 1.691a6.462 6.462 0 00-.954 1.232 8.24 8.24 0 00-.169.327c-.386.772-.579 1.626-.579 2.56 0 1.603.567 2.972 1.702 4.108.128.121.257.237.386.35z"/><radialGradient id="SVGID_4_" cx="15.459" cy="5.189" r="7.828" gradientTransform="matrix(1.0345 0 0 1.0354 -.648 10.826)" gradientUnits="userSpaceOnUse"><stop offset=".086" stop-color="#424ee9"/><stop offset=".447" stop-color="#58c9be"/><stop offset=".71" stop-color="#59cab5"/><stop offset="1" stop-color="#8ce785"/></radialGradient><path fill-rule="evenodd" clip-rule="evenodd" fill="url(#SVGID_4_)" d="M12.941 8.467a6.993 6.993 0 00-.084 1.111c0 .169.008.334.024.496 0 .121.008.241.024.362.177 1.562.845 2.923 2.004 4.083.394.396.813.733 1.255 1.015l-1.557 1.934c-.185-.162-.37-.331-.555-.508a7.52 7.52 0 01-1.159-1.51c-.596-1.024-.893-2.176-.893-3.456 0-.218.008-.431.024-.64a6.596 6.596 0 01.917-2.887z"/></svg>
```

Comment:
I see some id attributes using `SVGID_2_` pattern. Can we follow `pi-swish` name convention and convert it?

It could be `pi-swish-radiant-2`

Reply:
Hi, I have updated the swish id attributes. Please check and let us know. Thank you.